evaluator.py

```diff
@@ -52,7 +52,7 @@ def set_model(self, model: ExchangeModel):
     def get_callback(self):
         return self._callback
 
-    def plot_result(self, genotype, ax):
+    def plot_result(self, genotype, ax, show_roi_and_mdd=False):
```

Comment:
W sumie chyba bardziej czytelnie byłoby skopiować kontent tej funkcji do`model_loader.py`, bo nie potrzeba callback'ów ani innych funkcjonalności z tej klasy, a to trochę poprawi wygląd w main'ie. (trzeba będzie dodać nowe argumenty do tej funkcji jak data i model).